installer.sh

```diff
@@ -9,81 +10,96 @@ PROJECT_NAME=gbw
 BUILD_DIR=$(pwd)/.build
 BUILD_TARGET=""
 INSTALL_DIR=$(pwd)/.dist
-PACKAGE_DIR=$(pwd)/.packages
+PACKAGES_DIR=$(pwd)/.packages
+
+BROWSER_OPTIONS="<chromium|webkit|mswebview2>"
+BROWSER_DEFAULT="chromium"
 
-MSWEBVIEW_NUGET_V=1.0.3351.48
-CHROMIUM_NUGET_V=138.0.17
-BROWSER_OPTIONS="<chromium|mswebview2>"
 SYS_OPTS=""
 
 PATH=$(cygpath -w /ucrt64/bin):$PATH
 export PATH
+
 source ./.scripts/util.sh
-# For Windows, check that we are running in a MSYS2 UCRT environment
-check_env
+source ./.scripts/packages.sh
```

Comment:
"utils.sh" was getting a bit noisy, so I pulled out the package install specific stuff into "packages.sh"